infrastructure/terraform/accounts/digirati.tf

```
@@ -0,0 +1,83 @@
+# Roles
+
+resource "aws_iam_role" "dds_access" {
+  name               = "dds-access"
+  assume_role_policy = data.aws_iam_policy_document.dds_assume_role.json
+}
+
+# Assume role policies
+
+resource "aws_iam_role_policy" "dds_access_get_s3" {
+  role   = aws_iam_role.dds_access.name
+  policy = data.aws_iam_policy_document.archive_get.json
+}
+
+# Policy documents
+
+data "aws_iam_policy_document" "dds_assume_role" {
+  statement {
+    effect = "Allow"
+
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "AWS"
+      identifiers = [local.dds_principal_arn]
+    }
+  }
+}
+
+data "aws_iam_policy_document" "allow_external_export-bagit_access" {
```

```suggestion
data "aws_iam_policy_document" "allow_intranda_export_bucket_access" {
```

to match the names in the `resources` block.